docs/test-plan.md

```diff
@@ -57,14 +57,16 @@ choose to go through some of the Optional Test Cases.
 2. Select the `google/brotli` database (or download it if you don't have one already)
 3. Run a local query.
 4. Once the query completes:
+   - Chose the `#select` result set from the drop-down
```

Comment:
I wonder if we want this when it's essentially working around a ~bug~ unintended feature. I suggested on the bug issue that maybe we don't want to use the `FunLinesOfCode` query in the test plan.

Though on balance it doesn't really hurt to add this step in, so I don't actually have a problem with it.

Reply:
Let's continue the discussion on the bug issue. I'll get this in for now as it's an improvement. We can decide whether to change the behaviour and/or update the test plan there.

Reply:
As noted on the other issue, I'm not trying to block the PR. We can always change the testplan again if we want to.